pkgs/by-name/te/tetrio-desktop/package.nix

```diff
@@ -91,4 +69,4 @@ stdenv.mkDerivation (finalAttrs: {
     maintainers = with lib.maintainers; [ wackbyte ];
```

Comment:
Would you be interested in adding yourself as a maintainer?

Reply:
Yeah sure, I probably won't be doing many huge changes like this often, but I'd be down to contribute and follow any changes.

Reply:
Alright :)